manual/sphinx/user_docs/differential_operators.rst

```diff
@@ -430,7 +430,7 @@ is the background *equilibrium* magnetic field.
 |                  | derivatives. The `Laplacian` solver performs the   |
 |                  | inverse operation                                  |
 +------------------+----------------------------------------------------+
-| ``brackets``     | Poisson brackets. The Arakawa option, neglects the |
+| ``bracket``     | Poisson brackets. The Arakawa option, neglects the |
```

Comment:
Does it need another space to line up `|` bars? Not sure how picky RST is.

Reply:
Also good spot! Readthedocs builds the PR now, so we'll be able to check before merging

Reply:
> Does it need another space to line up | bars? Not sure how picky RST is.

Good spot, could do with that even if RST doesn't complain...
```suggestion
| ``bracket``      | Poisson brackets. The Arakawa option, neglects the |
```

Reply:
...and in the PR docs build, the table disappeared, so I guess the space was necessary.